test/Microsoft.ML.CodeGenerator.Tests/TransformGeneratorTests.cs

```diff
@@ -175,7 +175,7 @@ public void ImageLoadingTest()
             Pipeline pipeline = new Pipeline(new PipelineNode[] { node });
             CodeGenerator codeGenerator = new CodeGenerator(pipeline, null, null);
             var actual = codeGenerator.GenerateTransformsAndUsings(new PipelineNode[] { node });
-            string expectedTransform = "LoadRawImageBytes(\"Label\", null, \"Label\")";
+            string expectedTransform = "LoadImages(\"Label\", null, \"Label\")";
```

Comment:
Were there any parts of the CodeGen code in the [AutoML branch](https://github.com/dotnet/machinelearning/tree/features/automl) that are not being moved over? Or is this all of the CodeGen code?

_(left as a code comment to allow for a threaded discussion)_

Reply:
It's all of CodeGen code